2022-02-23 18:08:44,579 INFO    MainThread:6450 [wandb_setup.py:_flush():71] setting env: {}
2022-02-23 18:08:44,579 INFO    MainThread:6450 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-23 18:08:44,579 INFO    MainThread:6450 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220223_180844-6vf8h0lt/logs/debug.log
2022-02-23 18:08:44,579 INFO    MainThread:6450 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220223_180844-6vf8h0lt/logs/debug-internal.log
2022-02-23 18:08:44,579 INFO    MainThread:6450 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fd644b1f090>
2022-02-23 18:08:44,580 INFO    MainThread:6450 [wandb_init.py:init():404] calling init triggers
2022-02-23 18:08:44,580 INFO    MainThread:6450 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-23 18:08:44,580 INFO    MainThread:6450 [wandb_init.py:init():460] starting backend
2022-02-23 18:08:44,580 INFO    MainThread:6450 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 18:08:44,642 INFO    MainThread:6450 [backend.py:ensure_launched():216] starting backend process...
2022-02-23 18:08:44,701 INFO    MainThread:6450 [backend.py:ensure_launched():222] started backend process with pid: 6488
2022-02-23 18:08:44,704 INFO    MainThread:6450 [wandb_init.py:init():469] backend started and connected
2022-02-23 18:08:44,748 INFO    MainThread:6450 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-23 18:08:44,749 INFO    MainThread:6450 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-23 18:08:44,749 INFO    MainThread:6450 [wandb_init.py:init():533] updated telemetry
2022-02-23 18:08:44,856 INFO    MainThread:6450 [wandb_init.py:init():563] communicating current version
2022-02-23 18:08:46,597 INFO    MainThread:6450 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-23 18:08:46,598 INFO    MainThread:6450 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-23 18:08:46,819 INFO    MainThread:6450 [wandb_init.py:init():606] starting run threads in backend
2022-02-23 18:08:51,830 INFO    MainThread:6450 [wandb_run.py:_console_start():1810] atexit reg
2022-02-23 18:08:51,831 INFO    MainThread:6450 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-23 18:08:51,831 INFO    MainThread:6450 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-23 18:08:51,832 INFO    MainThread:6450 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-23 18:08:51,832 INFO    MainThread:6450 [wandb_init.py:init():633] run started, returning control to user process
2022-02-23 18:17:04,708 INFO    MainThread:6450 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 18:17:04,709 INFO    MainThread:6450 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 18:21:35,330 INFO    MainThread:6450 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-23 18:21:35,331 INFO    MainThread:6450 [wandb_run.py:_restore():1752] restore
2022-02-23 18:21:35,511 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-23 18:21:35,720 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-23 18:21:36,203 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25247
}

2022-02-23 18:21:36,307 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25247
}

2022-02-23 18:21:36,420 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25247
}

2022-02-23 18:21:36,523 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-23 18:21:36,628 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-23 18:21:36,730 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-23 18:21:36,832 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-23 18:21:36,935 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-23 18:21:37,433 INFO    MainThread:6450 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

